packages/terra-paginator/src/Paginator.jsx

```diff
@@ -120,22 +119,18 @@ class Paginator extends React.Component {
     return pageButtons;
   }
 
-  hasNavContext() {
-    return this.props.totalCount && this.props.itemCountPerPage;
-  }
-
   defaultPaginator() {
     const theme = this.context;
     const { intl } = this.props;
```

Comment:
Can you pull totalCount out to remove the need for this.props.totalCount throughout?

Reply:
updated [here](https://github.com/cerner/terra-core/pull/3190/commits/d0e5ef41693dba34291910c0e38aeb195fc36a43#diff-febb1905307240e51a8c9d0033fb21131b12efbdeeb6f75a5ceb94b8da70f486)